### DISCUSSION TITLE
Import to access user auth role

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#968] [Support / Q&A] [answered]
Hi @notsidney and @htuerker 
Any particular function that allows access to the user auth role.
{ userRolesAtom,currentUserAtom } from "@src/atoms/globalScope/auth";  is giving:

`
currentUserAtom: 
{init: undefined, debugLabel: 'currentUserAtom', toString: ƒ, read: ƒ, write: ƒ}
debugLabel
: 
"currentUserAtom"
init
: 
undefined
read
: 
ƒ ()
toString
: 
() => key
write
: 
(get, set, update) => {…}
[[Prototype]]
: 
Object
`

**Comment by RajGM** [comment]
@BeeBombshell

**Comment by RajGM** [comment]
How can we access the user auth role?

**Comment by RajGM** [comment]
Any updates @notsidney

**Comment by BeeBombshell** [comment, marked as answer]
Hey @RajGM, if this is still an issue, perhaps you can join our upcoming [office hours](https://discord.gg/vZbFQzje?event=1059820599184195604) so that we can go over this and get it resolved.

**Reply by RajGM** [reply]
Hi @BeeBombshell 
The issue is no longer relevant as the latest update has auth role from the controls settings.
